zh/backup-to-aws-s3.md

```diff
@@ -0,0 +1,513 @@
+---
+title: 使用 BR 工具备份 AWS 上的 TiDB 集群
+category: how-to
+---
+
+# 使用 BR 工具备份 AWS 上的 TiDB 集群
+
+本文详细描述了如何将运行在 AWS Kubernetes 环境中的 TiDB 集群数据备份到 AWS 的存储上。本文档中的“备份”，均是指全量备份（Ad-hoc 全量备份和定时全量备份）。底层通过使用 [`BR`](https://pingcap.com/docs-cn/v3.1/reference/tools/br/br) 获取集群的逻辑备份，然后再将备份数据上传到 AWS 的存储上。
+
+本文使用的备份方式基于 TiDB Operator v1.1 及以上版本的 Custom Resource Definition(CRD) 实现。
+
+## AWS 账号权限授予的三种方式
+
+在 AWS 云环境中，不同的类型的 Kubernetes 集群提供了不同的权限授予方式。本文测试了以下三种权限授予方式:
+
+1. 通过传入 AWS 账号的 AccessKey 和 SecretKey 进行授权:
```

Comment:
If the three methods are not in sequence, please use an unordered list.

Reply:
Fixed in https://github.com/pingcap/docs-tidb-operator/pull/99